patch-package
--- a/node_modules/enzyme-adapter-react-16/build/ReactSixteenAdapter.js
+++ b/node_modules/enzyme-adapter-react-16/build/ReactSixteenAdapter.js
@@ -56,6 +56,9 @@ var FunctionalComponent = 1;
 var HostPortal = 4;
 var HostComponent = 5;
 var HostText = 6;
+var Mode = 11;
+var ContextConsumer = 12;
+var ContextProvider = 13;
 
 function nodeAndSiblingsArray(nodeWithSibling) {
   var array = [];
@@ -111,9 +114,6 @@ function toTree(vnode) {
         instance: node.stateNode,
         rendered: childrenToTree(node.child)
       };
-    case Fragment:
-      // 10
-      return childrenToTree(node.child);
     case FunctionalComponent:
       // 1
       return {
@@ -145,6 +145,11 @@ function toTree(vnode) {
     case HostText:
       // 6
       return node.memoizedProps;
+    case Fragment: // 10
+    case Mode: // 11
+    case ContextProvider: // 13
+    case ContextConsumer: // 12
+      return childrenToTree(node.child);
     default:
       throw new Error('Enzyme Internal Error: unknown node with tag ' + String(node.tag));
   }
